CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Contributing to Garble
+
+Thank you for your interest in contributing! Here are some ground rules:
+
+1. The tool's design decisions are in the [README](README.md)
+2. New features or major changes should be opened as an issue first
+3. Non-trivial contributions should be done in PRs with code review and CI
+4. Repo access can be given after someone has contributed a few times
+
+### Testing
+
+Just the usual `go test ./...`; many of the tests are in
+[testscript](https://godoc.org/github.com/rogpeppe/go-internal/testscript) under
+`testdata/script/`, which allows laying out files and shell-like steps to run as
+part of the test.
+
+Note that the tests do real builds, so they are quite slow; on an average
```

Comment:
Windows requires admin rights, without them there will be an error: 
```
\Temp\go-build293132094\b001\garble.test.exe $WORK\.bin\garble.exe: A required privilege is not held by the client.
```

Reply:
Agh. I don't have a windows machine to test this on, unfortunately. All I can say is that the tests pass on Windows on CI. If you want to send a PR to make the tests more portable to Windows, I'd be happy to review.

Unless you mean the benchmark. CI doesn't run the benchmark, so I don't know if it works on Windows. But again, no way for me to test.

Reply:
This is probably not an error, but a peculiarity of Windows development. I think it's worth mentioning in CONTRIBUTING.md (or just fix it, I'll see if it's possible.)

Reply:
Fixed https://github.com/mvdan/garble/pull/66 :)

Reply:
Nice, thanks!